.github/workflows/test-local.yml

```diff
@@ -16,7 +16,7 @@ jobs:
         uses: ./
         with:
           ethereum_url: https://goerli.infura.io
-          requested_network: 'goerli'
+          requested_network: 'arbitrum-goerli'
```

Comment:
Probably remove this, but it's just for testing purposes.

Reply:
reverted back to georli :)